test/browser/webdriver-sauce.conf.js

```diff
@@ -39,7 +39,7 @@ exports.config = {
   // and 30 processes will get spawned. The property handles how many capabilities
   // from the same test should run tests.
   //
-  maxInstances: 10,
+  maxInstances: 1,
```

Comment:
Why less?

Reply:
I've noticed reliability issues with Sauce Labs so I made the webdriver settings a bit more gentle. We have four "capabilities" so a max instance of 1 runs four tests at once.